object_relations/bin/object_relations.dart

```diff
@@ -0,0 +1,75 @@
+import 'package:realm_dart/realm.dart';
+
+import 'model.dart';
+
+void main(List<String> arguments) {
```

Comment:
Main doesn't exits and never complete when using 0.2.0-alpha. It woks perfect when dll is build from master. Are they different versions or it is something else? @blagoev @nielsenko

Reply:
There is a bug in 0.2.0-alpha regarding closing realms, that has been fixed in master. Be aware that it is important to close the realm, it does not happen automatically.